### PR TITLE
category page

### PR DESCRIPTION
**Pull request by mehulsatardekar**
**Added category Page**

Preview : https://deploy-preview-3--quizgrad.netlify.app/views/category

category

### CODE REVIEW

File: public/css/category.css [hunk 1]
```diff
@@ -0,0 +1,69 @@
...
+    padding: 5px;
+    background: var(--img-banner);
+    border-top-left-radius: 2px;
+    border-bottom-left-radius: 10px;
+
+}
+
+.qg-gap-5{
+    gap: 5rem;
+}
```
Comment (yaswanthmyneni) [inline public/css/category.css]: remove this class as you have not used it in your Html page. The remaining code looks good to me. Can be merged.

Comment (mehulsatardekar) [reply public/css/category.css]: @yaswanthmyneni  Thanks. made the changes as u suggested.

File: public/css/category.css [hunk 2]
```diff
@@ -0,0 +1,67 @@
...
+    --img-text:#f1f2f3;
+    --img-banner:#4F46E5;
+}
+
+.navbar-bg{
+    background: var(--navbar-bg);
+}
+
+.nav-zindex{
+    z-index: 9999;
```
Comment (Naman-Saxena1) [inline public/css/category.css:15]: You can keep this a little low.

File: public/css/category.css [hunk 3]
```diff
@@ -0,0 +1,67 @@
...
+}
+.img-category{
+    min-width: 200px;
+    width: 19.5rem;
+    height: 100%;
+    border-radius: 10px;
+    background: var(--img-bg);
+    cursor: pointer;
+    min-height: 15.625rem;
```
Comment (Naman-Saxena1) [inline public/css/category.css:24]: I am not sure if you need to use 3 decimal points precision. But if you need it, then it's fine.

**Review by Naman-Saxena1 — CHANGES_REQUESTED** [review]
Everything else looks fine. 👍 
Btw, nice UI

**Review by yaswanthmyneni — APPROVED** [review]

**Review by thenorthkun — APPROVED** [review]
Good to merge ✅

**Review by rajaranjanmr — APPROVED** [review]

**Pull request merged**